libs/shared/ui/src/lib/components/modals/modal-multi-confirmation/use-modal-multi-confirmation/use-modal-multi-confirmation.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import { act, renderHook } from '__tests__/utils/setup-jest'
```

Comment:
Prefer to use `@qovery/shared/util-tests` 

```suggestion
import { act, renderHook } from '@qovery/shared/util-tests'
```